src/gwc/src/main/java/org/geoserver/gwc/config/GeoserverXMLResourceProvider.java

```diff
@@ -154,7 +154,7 @@ private Resource findOrCreateConfFile() throws IOException {
         Resource xmlFile = findConfigFile();
 
         if (Resources.exists(xmlFile)) {
-            LOGGER.info("Found configuration file in " + configDirectory.path());
+            LOGGER.debug("Found configuration file in '" + configDirectory.path() + "'");
```

Comment:
```suggestion
            LOGGER.config("Found configuration file in '" + configDirectory.path() + "'");
```

Reply:
This one may be appropriate, I will test and report back.

Reply:
It was actually the other way around, this oone has to stay at debug.

I found that this was using the log4j 2 api directly and updated it to follow our logging standards.